app/code/Bulbulatory/Recomendations/view/frontend/web/js/view/checkout/summary/recommendation-discount.js

```diff
@@ -0,0 +1,25 @@
+define(
+    [
+       'jquery',
+       'Magento_Checkout/js/view/summary/abstract-total',
+       'Magento_Checkout/js/model/quote',
+       'Magento_Checkout/js/model/totals',
+       'Magento_Catalog/js/price-utils'
+    ],
+    function ($,Component,quote,totals,priceUtils) {
+        "use strict";
+        return Component.extend({
+            defaults: {
+                template: 'Bulbulatory_Recomendations/checkout/summary/recommendation-discount'
+            },
+            totals: quote.getTotals(),
+            isDisplayedRecommendationDiscountTotal : function () {
+                return true;
```

Comment:
I tak jak wcześniej :stuck_out_tongue: zniżkę wyświetlmy tylko wtedy, gdy ona jest. Można to sprawdzić podobnie jak w funkcji poniżej - pobrać segment `recommendation_discount` i sprawdzić, czy on istnieje/ma wartość i na tej podstawie zwrócić true/false.